### DISCUSSION TITLE
dashboardVisualizer & HTML Report Usage for Existing JMX

### DISCUSSION THREAD

**Discussion by BharadwajSamavedam** [abstracta/jmeter-java-dsl#115] [Q&A] [answered]
1) How can i use the dashboardVisualizer for DslTestPlan.fromJmx ?
2) How to generate HTML report for DslTestPlan.fromJmx?

**Comment by rabelenda** [comment, marked as answer]
Hello, that is an interesting question, thank you for asking!

So far we haven't thought about such possibility, but in fact there is nothing preventing us from providing such feature.

Maybe just providing a children() method for such test plan would allow for attaching additional components like the dashboard & resultsTree visualizers in addition to the html report.

We will explore it and let you know

Thanks

**Comment by BharadwajSamavedam** [comment]
Thank you for your reply.

Happy to know.

**Comment by rabelenda** [comment]
[Latest release](https://github.com/abstracta/jmeter-java-dsl/releases/tag/v0.56) should support your use case. 

Let us know if this is what you expected.

Regards
